pkgs/racket-doc/scribblings/reference/promise.scrbl

```diff
@@ -173,3 +173,14 @@ TODO: Say something on:
   demand.
 ;}
 }
+
+@defform[(delay* body ...+)]{
+Like @racket[delay], except that it only handles one value, and it
+doesn't deal with exceptions.
+
+If the last @racket[body] expression either produces more than one
+value or raises an exception, forcing it for the first time will
+raise an exception, and subsequent forces will raise a "reentrant
```

Comment:
Can you make it more specific about what exception is raised in each case, the way the rest of the docs do?